src/app.js

```diff
@@ -1 +1,43 @@
 'use strict';
+
+const { generateRandomDigit } = require('./modules/generateRandomDigit');
+const { calculateBullsAndCows } = require('./modules/calculatingBullsAndCows');
+const { userInterface } = require('./modules/userInterface');
+
+const computerDigits = generateRandomDigit();
+
+const isValidDigits = (digits) => {
+  if (digits.length !== 4) {
+    return false;
+  }
+
+  const digitsArr = digits.split('');
+  const uniqueDigits = [...new Set(digitsArr)];
+
+  return uniqueDigits.length === digitsArr.length;
+};
+
+const question = () => {
+  userInterface.question('Enter 4 unique digits: ', (userDigits) => {
+    if (userDigits === computerDigits) {
+      // eslint-disable-next-line no-console
```

Comment:
You can use `/* eslint-disable no-console */` not to do it before every console.log